packages/ui/src/common/components/forms/ToggleCheckbox.tsx

```diff
@@ -14,19 +14,28 @@ export interface Props {
   onChange: (value: boolean) => void
   trueLabel: ReactNode
   falseLabel: ReactNode
+  switchToggle?: boolean
 }
 
-export function ToggleCheckbox({ isRequired, disabled, checked, onChange, trueLabel, falseLabel }: Props) {
+export function ToggleCheckbox({
+  isRequired,
+  disabled,
+  checked,
+  onChange,
+  trueLabel,
+  falseLabel,
+  switchToggle,
+}: Props) {
   const onClick = (setValue: boolean) => () => {
     if (disabled !== true) {
       onChange(setValue)
     }
   }
-
+  const toggleValidation = switchToggle ? switchToggle : false
   return (
     <ToggleContainer groupDisabled={disabled}>
       <ToggleLabel onClick={onClick(true)}>{trueLabel}</ToggleLabel>
-      <ToggleStyledInput isChecked={checked}>
+      <ToggleStyledInput isChecked={checked} switchToggle={toggleValidation}>
```

Comment:
What about something like that:
```ts
  hasNoOffState = false,
}: Props) {
  const onClick = (setValue: boolean) => () => {
    if (disabled !== true) {
      onChange(setValue)
    }
  }
  return (
    <ToggleContainer groupDisabled={disabled}>
      <ToggleLabel onClick={onClick(true)}>{trueLabel}</ToggleLabel>
      <ToggleStyledInput isChecked={checked} hasNoOffState={hasNoOffState}>
```
I'm not sure so convince by `hasNoOffState` but it's more inline with the other flag names in Pioneer, and  using `switchToggle` and the extra `toggleValidation` is kind of confusing